apps/wallet-ui/src/app/[ecosystem]/login/page.tsx

```diff
@@ -0,0 +1,27 @@
+"use client";
+import { client } from "@/lib/client";
+import { useTheme } from "next-themes";
+import { ConnectEmbed } from "thirdweb/react";
+import { ecosystemWallet } from "thirdweb/wallets";
+import { generatePayload, isLoggedIn, login, logout } from "./auth";
+
+export default function Page({ params }: { params: { ecosystem: string } }) {
+  const { theme } = useTheme();
+
+  return (
+    <ConnectEmbed
+      theme={theme as "light" | "dark"}
```

Comment:
instead of casting it you can do a ternary on `theme`



```suggestion
      theme={theme === "light" ? "light" : "dark"}
```